2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -8,7 +8,23 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+
+  div.style.cssText = `
+    height: 100vh; 
+    font-size: 40px; 
+    display: flex; 
+    justify-content: center; 
+    align-items: center`;
+
+  function currentTimeUpdate (){
```

Comment:
Quick tip: although this code is valid, if you start using "linters" (tools that check the validity of your code and how it is written) then it will complain about the space between the name of the function and the parentheses.

```suggestion
  function currentTimeUpdate() {
```